.env.base

```diff
@@ -169,3 +171,7 @@ REACT_APP_ZERION_BASE_URL=https://api.proxy.shapeshift.com/api/v1/zerion
 
 # 0x
 REACT_APP_ZRX_BASE_URL=https://api.proxy.shapeshift.com/api/v1/zrx/
+
+# chainflip
+REACT_APP_CHAINFLIP_API_KEY=6ba154d4-e219-472a-9674-5fa5b1300ccf
```

Comment:
Give me a ping to set up Shapeshift's API key when going forward, this is currently mine for testing! :) The best person to reach out to me for this would be the one who will be responsible for redeeming affiliate rewards in the future. The one controlling the wallet to sign Chainflip transactions that is (redeeming is a chainflip tx)

Reply:
Sounds good @CumpsD! Is there any reason we should wait for go-live to get an actual API key here, or can we already get one? Will get the convs going already

Reply:
The key in there right now is actually the one from my swapping Discord bot https://swappy.be/ :) It works, you can test with it, but it charges 0.15% fee (0.05 BaaS + 0.10 Swappy). I can make you a temp one that does only 0.05 (BaaS fee + 0% partner) if you want. You can go live with that as well and swap it out to the Shapeshift one afterwards. lmk

Reply:
@CumpsD just to be sure, we can parametrize fee bps right i.e this isn't tied to the API key used?

Reply:
Current one for dev is absolutely fine, so long as we go live on prod (and ideally on develop as a follow-up) with a proper API key 🙏🏽

Reply:
> @CumpsD just to be sure, we can parametrize fee bps right i.e this isn't tied to the API key used?

Yes, you can pass in `commissionBps` for both quote and opening a deposit channel (I think I added that in the PR already even! taking in affiliateBps from shapeshift)